### ISSUE TITLE
ESP32 debugger not working

### ISSUE THREAD

**Issue by gidagel** [Moddable-OpenSource/moddable#662]
**Build environment:** Linux Ubuntu 
**Target device:** ESP32

**Description**
Hi, I have an ESP32 and I'm being able to program it in Arduino normally. However, I installed Moddable on Ubuntu, it compiles, writes on the board, but apparently it doesn't work (I tested it with Blink Example and the LEDs didn't blink), also nothing appears in the debugger. Below is the screenshot of the current state. 

**Expected behavior**
I expected the board to blink and to see the xsdebug working.

**Images**
![image](https://user-images.githubusercontent.com/50461522/124172456-2978a500-daaa-11eb-89d0-467c2d90a1c1.png)

**Comment by mkellner** [comment]
I just did a fresh install on Ubuntu 20.04.1 without problems.

Ensure you are up to date with ESP IDF. Here is what my install looked like:

![image](https://user-images.githubusercontent.com/6822704/124191227-5ad87d00-da78-11eb-98c8-fa17ff1c7bec.png)

Follow the install instructions for linux including the
`cd $IDF_PATH
./install.sh`

What was the build line that you used? Did the platform (`-p` option) match your device?

I did a build for a Moddable Two
`mcconfig -d -m -p esp32/moddable_two`
and for a raw esp32 chip with
`mcconfig -d -m -p esp32/nodemcu`
or
`mcconfig -d -m -p esp32`
